docs/ml-guide.md

```diff
@@ -655,6 +660,36 @@ import org.apache.spark.sql.DataFrame;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SQLContext;
 
+// Labeled and unlabeled instance types.
```

Comment:
Yes it's annoying when copy/pasting a bunch of code into spark shell and it fails because these classes are not declared.